fontbe/src/glyphs.rs

```diff
@@ -22,6 +29,86 @@ pub fn create_glyph_work(glyph_name: GlyphName) -> Box<BeWork> {
     Box::new(GlyphWork { glyph_name })
 }
 
+fn create_component(
+    context: &Context,
+    ref_glyph_name: &GlyphName,
+    transform: &Affine,
+) -> Result<(Component, Bbox), GlyphProblem> {
+    // Obtain glyph id from static metadata
+    let gid = context
+        .ir
+        .get_static_metadata()
+        .glyph_id(ref_glyph_name)
+        .ok_or(GlyphProblem::NotInGlyphOrder)?;
+    let gid = GlyphId::new(gid as u16);
+
+    // No known source does point anchoring so we just our transform into a 2x2 + offset
+    let [a, b, c, d, e, f] = transform.as_coeffs();
+    let component = Component::new(
+        gid,
+        Anchor::Offset {
+            x: e as i16,
+            y: f as i16,
+        },
+        Transform {
+            xx: F2Dot14::from_f32(a as f32),
+            yx: F2Dot14::from_f32(b as f32),
+            xy: F2Dot14::from_f32(c as f32),
+            yy: F2Dot14::from_f32(d as f32),
+        },
+        ComponentFlags::default(),
```

Comment:
FWIW, ufo2ft defaults to 0x4 ROUND_XY_TO_GRID. It also sets USE_MY_METRICS to the first component whose base glyph's advance width is equal to the composite glyph's advance width.

Reply:
also note, support for additional glyf flags set from UFO public.truetype.* lib keys will be added with https://github.com/googlefonts/ufo2ft/pull/654

Reply:
https://github.com/googlefonts/fontmake-rs/issues/136 for the new flags

Reply:
I set round xy to grid.  I haven't done anything with metrics yet so I've pushed that to https://github.com/googlefonts/fontmake-rs/issues/137. I should probably generate an hmtx at some point I suppose.